data/archive/nwerc/1989.yaml

```diff
@@ -1,6 +1,6 @@
 contest:
-  name: NWERC 1989
-  date: Eindhoven
+  name: ERC 1989
```

Comment:
But then why does the problem set say "Northwestern European Regionals"? :thinking: 

![image](https://user-images.githubusercontent.com/9739541/221365619-566b68a1-a427-4741-bbbc-59fb8570383e.png)

Reply:
This is actually due to the old CHipCie typing over the problem set. The quality is quite bad (it's a copy of a copy), so i kept the remade files, even if they use the wrong name.
![IMG20230226075714](https://user-images.githubusercontent.com/5221158/221396897-36d84088-8bfa-41b1-9e8e-1440d6feadcb.jpg)

Reply:
I've updated the headers in this `problemset.pdf` (same for 1992) to make this more consistent :smile: